dynamic_env_pytest_tests/tests/services/http_gate/test_http_headers.py

```diff
@@ -67,7 +77,7 @@ def storage_objects_with_attributes(self, request: FixtureRequest) -> list[Stora
             storage_object_id = upload_via_http_gate_curl(
                 cid=cid,
                 filepath=file_path,
-                endpoint=f"http://{self.neofs_env.http_gw.address}",
+                endpoint=f"http://{self.neofs_env.http_gw.address}",  # ???,
```

Comment:
When you tried to interrupt those frozen tests, what was the traceback? Did it fail on REST or HTTP? From what I see, it can be a bug with the rest gateway.

Reply:
Yep, you're right it relates to bad request response due to "@" symbols in attributes.